### PR TITLE
refactor: sending alerts to sqs instead of logging

### PR DESCRIPTION
**Pull request by andreabadesso**
### Acceptance Criteria
- Alerts should be send directly to the alerts SQS instead of just logging them

### Security Checklist
- [X] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: .eslintrc.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
+parser: "@typescript-eslint/parser"
```
Comment (andreabadesso) [inline .eslintrc.yml:1]: *Comment unrelated to line number*

This is the `eslintrc` we are using on the Wallet Service

**Comment by luislhl** [comment]
I"ve talked to Abadesso on Slack about this.

Maybe we should make this through our log-manager solution, because it has mechanisms intended to avoid spam in our channels and in our alert queue.

But to do this, we would need to log in JSON here

**Review by r4mmer — APPROVED** [review]
Many uses (with `Severity.INFO`) seem like they're not actually an alert so I think another name (e.g. `sendLog`) would be better for the `addAlert`.

**Comment by luislhl** [comment]
> I've talked to Abadesso on Slack about this.
> 
> Maybe we should use our log-manager solution for this, because it has mechanisms intended to avoid spam in our channels and in our alert queue.
> 
> But to do this, we would need to log in JSON here

After further discussion, this will be done in a new issue, and we will proceed with this PR

File: src/api/lambda.ts [hunk 2]
```diff
@@ -67,6 +67,43 @@ export const lambdaCall = (fnName: string, payload: any): Promise<any> =>
...
+    severity,
+    metadata,
+    application: process.env.APPLICATION_NAME,
+  };
+
+  const sqs = new AWS.SQS({ apiVersion: '2015-03-31' });
+
+  const params = {
+    MessageBody: JSON.stringify(preparedMessage),
+    QueueUrl: process.env.ALERT_QUEUE_URL as string,
```
Comment (luislhl) [inline src/api/lambda.ts:90]: Just remember we will need to add this in our infra code as well

File: src/api/lambda.ts [hunk 3]
```diff
@@ -67,6 +67,43 @@ export const lambdaCall = (fnName: string, payload: any): Promise<any> =>
...
+      None: {
+        DataType: 'String',
+        StringValue: '--',
+      },
+    }
+  };
+
+  sqs.sendMessage(params, (err) => {
+    if (err) {
+      logger.error('[ALERT] Erroed while sending message to the alert sqs queue');
```
Comment (luislhl) [inline src/api/lambda.ts:101]: This means we shouldn't disable the old alerting mechanism yet, right?

**Review by luislhl — APPROVED** [review]

File: src/api/lambda.ts (hunk 3 above)
Comment (andreabadesso) [reply src/api/lambda.ts:101]: That is correct, a failure to create the alert should be an alert
